shiny/ui/_page.py

```diff
@@ -466,7 +544,7 @@ def page_bootstrap(
     head = tags.title(title) if title else None
     return tags.html(
         tags.head(head),
-        tags.body(*bootstrap_deps(), *args, **kwargs),
+        tags.body(*bootstrap_theme_deps(theme), *args, **kwargs),
```

Comment:
Since this is now a `TagList` instead of a `list`, it's probably better not to have the `*`?

```suggestion
        tags.body(bootstrap_theme_deps(theme), *args, **kwargs),
```